README.md

```diff
@@ -27,17 +27,53 @@ Designed for [Shields][] though may be useful for other application as well.
 [shields]: https://github.com/badges/shields
 [convert-svg-to-png]: https://www.npmjs.com/package/convert-svg-to-png
 
+## Architecture
+
+- The code is is [one file][].
+- It's based on the [Micro][] framework.
+- Rasterization uses headless Chromium via [Puppeteer][], wrapped in the
+  [convert-svg-to-png][] library.
+- The service has no state. It simply fetches corresponding SVG badges from the
+  upstream service, rasterizes them, and forwards cache headers along with the
+  response.
+- Caching the rasterized badges is handled by a downstream CDN.
+
+[one file]: https://github.com/badges/svg-to-image-proxy/blob/master/rasterize.js
+[micro]: https://github.com/zeit/micro
+[puppeteer]: https://pptr.dev/
+
 ## Deployment
 
 Runs on Zeit Now.
 
-Staging is deployed automatically from master: https://shields-raster-staging.now.sh/
+## raster.shields.io production hosting
+
+Staging: https://shields-raster-staging.now.sh/
+
+Production: https://raster.shields.io/
+
+`raster.shields.io` hosting is provided by:
+
+- [Zeit Now][zeit now]
+- [Zeit Smart CDN][cdn]
+- [Now for GitHub][]
+
+Pull requests are deployed automatically. Staging is deployed automatically
+from master. Production is deployed manually for now.
+
+For information about current deploys, see the [Now dashboard][].
 
-Production is deployed manually for now.
+To deploy, run `now alias shields-raster-staging.now.sh raster.shields.io`.
+This ships the current staging build to production.
 
-To deploy, run `now alias shields-raster-staging.now.sh raster.shields.io`
+`raster.shields.io` is configured in the Shields Cloudflare as a CNAME for
+`alias.zeit.co` with traffic configured not to pass through Cloudflare. That's
+because it's cached by [Now's Smart CDN][cdn] instead.
 
-The production server is at: https://raster.shields.io/
+[zeit now]: https://zeit.co/now
+[cdn]: https://zeit.co/smart-cdn
+[now for github]: https://zeit.co/github
+[now dashboard]: https://zeit.co/shields1/svg-to-image-proxy
```

Comment:
Who has access to this dashboard? I got "404: We could not find the resource or you do not have permissions to access it." at this page. Is access is limited I suggest to add a proper info about it.

Reply:
I added Chris and Caleb (badges/shields#3643) though I'm happy to invite you too.

At some point we should address this question more systematically: badges/shields#2577

Reply:
Thanks. I have an access now.